src/components/Block/Votes.vue

```diff
@@ -3,48 +3,52 @@ import { ref, computed, watch, toRefs } from 'vue';
 import { getChoiceString } from '@/helpers/utils';
 import { useProfiles } from '@/composables/useProfiles';
 import { useWeb3 } from '@/composables/useWeb3';
+import { clone } from '@snapshot-labs/snapshot.js/src/utils';
```

Comment:
Off-topic: Don't we want to just do `import snapshot from '@snapshot-labs/snapshot.js'` and then `snapshot.utils.clone()`?
I think snapshot.js is imported like this in other places anyway, in some plugins at least I think, and reading `snapshot.utils.clone()` is just much more clear than always having to check where a function comes from.

Could also be added as a Vue plugin.

Reply:
Hm, just `clone` seems fine for me. I also couldn't find any other places we use snapshot.js like that.

Reply:
Then it was maybe only in a plugin.